src/Mocks/VisualStudio/project.15.0.json

```diff
@@ -1,21 +1,21 @@
 ﻿{
   "dependencies": {
     "MicroBuild.Core": "0.2.0",
-    "Microsoft.VisualStudio.CoreUtility": "15.0.25824-RC",
+    "Microsoft.VisualStudio.ComponentModelHost": "15.0.25824-RC",
     "Microsoft.VisualStudio.Editor": "15.0.25824-RC",
     "Microsoft.VisualStudio.ImageCatalog": "15.0.25824-RC",
-    "Microsoft.VisualStudio.ProjectSystem": "15.0.561-pre-g30afd8d890",
+    "Microsoft.VisualStudio.ProjectSystem": "15.0.594-pre",
     "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": "14.3.25407",
     "Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime": "15.0.25726-Preview5",
     "Microsoft.VisualStudio.SDK.EmbedInteropTypes": "14.1.4",
-    "Microsoft.VisualStudio.Text.Data": "15.0.25824-RC",
-    "Microsoft.VisualStudio.Text.Logic": "15.0.25824-RC",
-    "Microsoft.VisualStudio.Text.UI": "15.0.25824-RC",
-    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.25824-RC",
+    "Microsoft.VisualStudio.Text.Data": "15.0.25901-RC",
+    "Microsoft.VisualStudio.Text.Logic": "15.0.25901-RC",
+    "Microsoft.VisualStudio.Text.UI": "15.0.25901-RC",
+    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.25901-RC",
     "Microsoft.VisualStudio.TextManager.Interop.11.0": "11.0.61030",
     "Microsoft.VisualStudio.TextManager.Interop.12.0": "12.0.30110",
     "NSubstitute": "1.10.0",
-    "System.Collections.Immutable": "1.3.0",    
+    "System.Collections.Immutable": "1.2.0",
```

Comment:
This thing is weird. I tried 1.3.0 got bunch of build issues. Looks like current (RC2) packages are bound to 1.2 so I changed all to 1.2 for consistency.